### PR TITLE
DOC: clarify the definition of the pdf of `stats.fisk`

### PR DESCRIPTION
**Pull request by Mazensayed91**

### CODE REVIEW

**Comment by Mazensayed91** [comment]
This related to #13996 issue.
I just added another definition for the fisk pdf

**Review by tupui — COMMENTED** [review]
Thanks @Mazensayed91. Although per Matt's comment, we should switch the two formulas. Have the one without the minus as the default one.

Also there is a comment about this being also relevant for Burr. If this is correct, we could fix this here as well.

**Comment by Mazensayed91** [comment]
Thanks @tupui for the modifications, I've modified the request to fit them

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -883,6 +883,12 @@ class burr_gen(rv_continuous):
...
 
+    .. math::
+
+        f(x, c, d) = c d x^{c - 1} / (1 + x^{c})^{d + 1}
+        
+    Also equivalent to:
+
     .. math::
 
         f(x, c, d) = c d x^{-c - 1} / (1 + x^{-c})^{d + 1}
```
Comment (ev-br) [inline scipy/stats/_continuous_distns.py:894]: These are only equivalent for `d=1`, i.e. for the fisk distribution.

Comment (Mazensayed91) [reply scipy/stats/_continuous_distns.py:894]: This is the pdf formula for burr distribution, so the first one is the right one ?

image

Comment (rgommers) [reply scipy/stats/_continuous_distns.py:894]: I agree, looks like the first one is the right one. Also, if we're using LaTeX anyway, let's use `\frac` and make it look identical to how the pdf is given on Wikipedia.

File: scipy/stats/_continuous_distns.py [hunk 2]
```diff
@@ -1079,10 +1085,17 @@ class fisk_gen(burr_gen):
     -----
     The probability density function for `fisk` is:
 
+    .. math::
+    
+        f(x, c) = c x^{c-1} (1 + x^{c})^{-2}
+
+    Also equivalent to:
```
Comment (rgommers) [inline scipy/stats/_continuous_distns.py:1092]: This is actually confusing, without reading the linked issue it was non-obvious that this wasn't a mistake. I suggest the following rephrase: _Please note that the above expression can be transformed into the following one, which is also commonly used:_

File: scipy/stats/_continuous_distns.py (hunk 1 above)
Comment (mdhaber) [reply scipy/stats/_continuous_distns.py:894]: image

Comment (mdhaber) [reply scipy/stats/_continuous_distns.py:894]: The second one, the original one, is the intended one for the `burr` distribution. 

> This is the PDF corresponding to the third CDF given in Burr's list;
    specifically, it is equation (11) in Burr's paper [1]_

This is:
image

Taking the derivative and changing the symbols used for the parameters, it turns into:
image

**Comment by mdhaber** [comment]
In case others would like to check, here are comparisons of how the documentation now renders VS Wikipedia.

<details>

### Fisk
[_Wikipedia (log-logistic distribution)_](https://en.wikipedia.org/wiki/Log-logistic_distribution)
image

[_SciPy_ (`fisk`)](https://43657-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.stats.fisk.html#scipy.stats.fisk)
image

### Burr Type III
[_Wikipedia (Dagum distribution)_](https://en.wikipedia.org/wiki/Dagum_distribution)
image

It is not obvious how the CDF would turn into the PDF above or how the PDF shown here would be equivalent to the PDF in SciPy. (Sympy cannot confirm it, so I'm not even sure if the PDF on Wikipedia is correct.) But it is more obvious that the derivative of the CDF above, which is the same as that in Burr's paper, is equivalent to the PDF used in SciPy.
```python3
from sympy import symbols, diff, simplify
x, a, p, c, d = symbols('x, a, p, c, d', real=True, positive=True)
F = (1 + x**(-a))**(-p)  # Wikipedia CDF
f = diff(F, x).subs(a, c).subs(p, d)  # Wikipdia deriviative of CDF, SciPy Notation
f_s = c*d*(x**(-c-1))/(1+x**(-c))**(d+1)  # SciPy PDF
print(simplify(f - f_s))  # 0
```

[_SciPy_](https://43657-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.stats.burr.html#scipy.stats.burr)
image


### Burr Type XII

[_Wikipedia ("Burr Distribution")_](https://en.wikipedia.org/wiki/Burr_distribution)
image

[_SciPy_ (`burr12`)](https://43657-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.stats.burr12.html#scipy.stats.burr12)
image

I see that it says `burr` instead of `burr12` above; I'll fix that.

</details>

I'm going to go ahead and merge this, but please let me know if you notice anything that the updates did not catch.

**Pull request merged**